sources/qa-lang-main.php

```diff
@@ -145,10 +145,10 @@
        'selected'=>'izbrano', 
        'send_button'=>'Pošlji', 
        'since_x'=>'od ^', //since ^
-       'suggest_ask'=>'Pomagaj nam začeti in ^1asking question^2.', //Help get things started by ^1asking a question^2.
-       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions v tej category^2.', //To see more, click for all the ^1questions in this category^2.
-       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2.', //To see more, click for the ^1full list of questions^2.
-       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
+       'suggest_ask'=>'Pomagaj nam začeti z ^1asking a question^2.', //Help get things started by ^1asking a question^2.
+       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions in this category^2.', //To see more, click for all the ^1questions in this category^2.
+       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full list of questions^2.', //To see more, click for the ^1full list of questions^2.
+       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full list of questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
```

Comment:
sem ponovno popravil te napake in naredil pull request kot vedno prej ko je
je delovalo, zdaj pa ne prikaže popravkov v 'skupni' mapi

Dne 13. april 2015 17.44 je Ambrož Vidergar <vidergar.ambroz@gmail.com>
napisal/-a:

> Popravek teh napak sem poslal že vsaj dvakrat, nevem zakaj tega ne prikaže?
> Bom poskusil še enkrat
>
> 2015-04-13 16:57 GMT+02:00 Matjaž M. <notifications@github.com>:
>
>> In sources/qa-lang-main.php
>> <https://github.com/matmr/q2a-prevajanje-FS/pull/9#discussion_r28245322>:
>>
>> > @@ -145,10 +145,10 @@
>> >         'selected'=>'izbrano',
>> >         'send_button'=>'Pošlji',
>> >         'since_x'=>'od ^', //since ^
>> > -       'suggest_ask'=>'Pomagaj nam začeti in ^1asking question^2.', //Help get things started by ^1asking a question^2.
>> > -       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions v tej category^2.', //To see more, click for all the ^1questions in this category^2.
>> > -       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2.', //To see more, click for the ^1full list of questions^2.
>> > -       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
>> > +       'suggest_ask'=>'Pomagaj nam začeti z ^1asking a question^2.', //Help get things started by ^1asking a question^2.
>> > +       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions in this category^2.', //To see more, click for all the ^1questions in this category^2.
>> > +       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full list of questions^2.', //To see more, click for the ^1full list of questions^2.
>> > +       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full list of questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
>>
>> Še vedno napaka na koncu.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/matmr/q2a-prevajanje-FS/pull/9/files#r28245322>.
>>
>
>